### PR TITLE
Notes Meeting 29042021

### PR DESCRIPTION
**Pull request by Bo016**
I worte some notes during the Meeting. Please complete those with your own (if existing) so that we dont forget the feedback or ideas.

### CODE REVIEW

**Review by timgrewe — APPROVED** [review]
This is just about what I also took away from the meeting.

**Review by fricklerhandwerk — CHANGES_REQUESTED** [review]
Translated to English and added notes on why/how to do hand-eye-calibration and how to drive the actual robot.

File: MeetingGerlach29042021.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+openCV nicht so ganz so super - eher eigene
+paper von Gerlach, upload abwarten
+
+Problemstellung Position Nadel:
+Tool calibrierung?
+commonly used: Pivotisierung
+
+Was dürfen wir benutzen:
+openCV gerne
+andere ROS nodes bitte nicht
```
[the remark this answers is not in the document]
Comment (fricklerhandwerk) [reply MeetingGerlach29042021.txt:26]: I also suggest to rename the file using [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) dates and to use [markdown](https://en.wikipedia.org/wiki/Markdown) for plain text documents generally: `2021-04-29 tutor meeting notes.md`.
ISO dates are unambiguous, sortable and (in my opinion) still easy to read. Markdown conveys a bit of structure while still being plain text, and can be nicely converted to other document types such as PDF or HTML when needed.

[the remark this answers is not in the document]
Comment (timgrewe) [reply MeetingGerlach29042021.txt:26]: I absolutely agree on the date formatting.

**Review by timgrewe — APPROVED** [review]

**Review by tobiastuhh — APPROVED** [review]
looks good

**Review by fricklerhandwerk — APPROVED** [review]

**Pull request merged**